docs/pages/enroll-resources/mcp-access/sse.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: MCP Access with SSE MCP Server
+sidebar_position: 4
+description: How to access MCP server with SSE transport.
+tags:
+- how-to
+- mcp
+- zero-trust
+---
+
+Teleport can provide secure access to MCP servers with SSE transport.
+
+This guides shows you how to:
+
+- Enroll a MCP server with SSE transport in your Teleport cluster.
+- Connect to the SSE MCP server via Teleport.
+
+## How it works
+
+![How it works](../../../img/mcp-access/architecture-sse.svg)
+
+Users can configure their MCP clients such as Claude Desktop to start an MCP
+server using `tsh`. Once successfully authorized, `tsh` establishes a session
+with the Application Service.
+
+The Application Service first starts a SSE connection to the remote MCP server
```

Comment:
Do we pronounce each letter of "SSE", or is it pronounced something like "see"? If it's the former, we'll want to say "an SSE connection". If it's the latter, "a SSE" is fine.